components/Api/Stability/index.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import Stability from './index';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+type Story = StoryObj<typeof Stability>;
+type Meta = MetaObj<typeof Stability>;
+
+export const Default: Story = {
+  args: {
+    stability: 0,
+    children: 'children',
```

Comment:
You could add some example test here instead of just "children" which is confusing

Reply:
Yeah!

Reply:
I don't found any thing to control children in same time of stability

Reply:
What? I think you misunderstood me. I'm just asking you to change the value of "children" to something more meangiful. For example:

```suggestion
    children: 'Insert the text of your Alert here',
```

Reply:
I mean : it's possible to change children in same time of stability?